test/image/mocks/gl_text_chart_arrays.json

```diff
@@ -0,0 +1,144 @@
+{
+    "data": [
```

Comment:
gl2d mocks use the `gl2d_` prefix

@dy do you mind renaming these new mocks?

Reply:
Fixed